### DISCUSSION TITLE
Season length in AutoMLForecast?

### DISCUSSION THREAD

**Discussion by concimuscb** [Nixtla/mlforecast#398] [Q&A]
What does the parameter do exactly here? What would be the right approach if we are forecasting multiple series each with different seasonalities? What if we do not know what is the season length?

Thanks in advance!

**Comment by jmoralez** [comment]
It's used to automatically generate a feature space. If you're providing one already you can set it to anything.

**Reply by concimuscb** [reply]
Thank you very much for the reply.

It might be interesting explicitly say that it is irrelevant if a feature space is provided. I understand it is somewhat implicit, but it is not 100% clear imo.

**Reply by jmoralez** [reply]
Yeah I think I'll make it optional and raise an error if `init_config` isn't provided

**Reply by jmoralez** [reply]
Done in #399
